modules/monitoring-modifying-retention-time-and-size-for-prometheus-metrics-data.adoc

```diff
@@ -37,113 +52,73 @@ If any data blocks exceed the defined retention time or the defined size limit,
 +
-[source,terminal]
+[source,terminal,subs="attributes+"]
 ----
-$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
+$ oc -n {namespace-name} edit configmap {configmap-name}
 ----
 
-.. Add the retention time and size configuration under `data/config.yaml`:
+. Add the retention time and size configuration under `data/config.yaml`:
 +
-[source,yaml]
+[source,yaml,subs="attributes+"]
 ----
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: cluster-monitoring-config
-  namespace: openshift-monitoring
+  name: {configmap-name}
+  namespace: {namespace-name}
 data:
   config.yaml: |
-    prometheusK8s:
-      retention: <time_specification> <1>
-      retentionSize: <size_specification> <2>
+    {component}:
+      retention: <time_specification> # <1>
+      retentionSize: <size_specification> # <2>
 ----
-+
 <1> The retention time: a number directly followed by `ms` (milliseconds), `s` (seconds), `m` (minutes), `h` (hours), `d` (days), `w` (weeks), or `y` (years). You can also combine time values for specific times, such as `1h30m15s`.
 <2> The retention size: a number directly followed by `B` (bytes), `KB` (kilobytes), `MB` (megabytes), `GB` (gigabytes), `TB` (terabytes), `PB` (petabytes), and `EB` (exabytes).
 +
-The following example sets the retention time to 24 hours and the retention size to 10 gigabytes for the Prometheus instance that monitors core {product-title} components:
-+
-[source,yaml]
-----
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: cluster-monitoring-config
-  namespace: openshift-monitoring
-data:
-  config.yaml: |
-    prometheusK8s:
-      retention: 24h
-      retentionSize: 10GB
-----
-
-** *To modify the retention time and size for the Prometheus instance that monitors user-defined projects*:
-endif::openshift-dedicated,openshift-rosa[]
-.. Edit the `user-workload-monitoring-config` `ConfigMap` object in the `openshift-user-workload-monitoring` project:
-+
-[source,terminal]
-----
-$ oc -n openshift-user-workload-monitoring edit configmap user-workload-monitoring-config
-----
-
-.. Add the retention time and size configuration under `data/config.yaml`:
-+
-[source,yaml]
-----
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: user-workload-monitoring-config
-  namespace: openshift-user-workload-monitoring
-data:
-  config.yaml: |
-    prometheus:
-      retention: <time_specification> <1>
-      retentionSize: <size_specification> <2>
-----
-+
-<1> The retention time: a number directly followed by `ms` (milliseconds), `s` (seconds), `m` (minutes), `h` (hours), `d` (days), `w` (weeks), or `y` (years).
-You can also combine time values for specific times, such as `1h30m15s`.
-<2> The retention size: a number directly followed by `B` (bytes), `KB` (kilobytes), `MB` (megabytes), `GB` (gigabytes), `TB` (terabytes), `PB` (petabytes), or `EB` (exabytes).
+The following example sets the retention time to 24 hours and the retention size to 10 gigabytes for the Prometheus instance:
```

Comment:
The 40 Gi mentioned in modules/monitoring-configuring-a-persistent-volume-claim.adoc has nothing to do with the 10 mentioned here?

Reply:
No, not really, the key here is to just add an example of resizing the storage down, because we cannot size it up. The example values do not really matter, but just for the consistency, the number chosen here for resizing is smaller that the number that is set in the example for initial configuring (if someone was about to follow the instrustions exactly as set in the docs).